charts/daemonset/values.yaml

```diff
@@ -0,0 +1,93 @@
+global:
+  # -- Cloud specific bindings (options: aws, gcp)
+  cloud: ""
+
+# -- Overrides the full-name given to the deployment resources (default: .Release.Name)
+fullnameOverride: ""
+
+image:
+  repository: "nginx"
```

Comment:
maybe switch to busybox for our artifacthub reputation?

Reply:
need a long running process to make the integration test for the chart work - could do busybox with a sleep I guess?

Reply:
I am not too bothered by it flagging up on ArtifcatHub tbh - when we produce app specific charts that fixes itself.